Pipfile

```diff
@@ -4,15 +4,15 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-responses = "==0.12.1"
+respx = "==0.16.3"
 
 [packages]
 beautifulsoup4 = ">=4.9.3"
 mako = ">=1.1.4"
 markupsafe = "==1.1.1"
-pysocks = ">=1.7.1"
+httpx-socks = "==0.3.1"
 pytest = ">=5.4.1"
-requests = ">=2.25.1"
+httpx = "==0.17.1"
```

Comment:
The docker build of the master branch is currently broken with the last brower-cookie3 version. Please check if we have no problem with this dependency.

Reply:
we had a build of the dockerfile on the github action.